### ISSUE TITLE
Symfony 2.6 framework bundle controller inheritance

### ISSUE THREAD

**Issue by slashfan** [scrutinizer-ci/php-analyzer#566]
Hi,

There seems to be a problem with the symfony2.6 controller inheritance. 
The new methods don't seem to be detected in my controlles and raise bug problems : addFlash(), redirectToRoute(), etc... 

Is there a way to fix that ? I'd prefer not to ignore those errors. 

Thanks !

see https://scrutinizer-ci.com/b/lexik/piggybee/issues/develop/files/src/AppBundle/Controller/AccountController.php?selectedSeverities%5B0%5D=10&orderField=path&order=asc&issueId=5e954e3b-ef21-464b-86fb-7bd167749f5a

**Comment by schmittjoh** [comment]
Did you update your composer.lock file already?

**Comment by slashfan** [comment]
Yes of course, in fact several of our team projects present the same false positive. 
Also, functional tests that cover those use case run perfectly on scrutinizer so I don't understand.
